changelogs/fragments/619-inventory-aws_ec2-assume-arn-role-when-listing-regions.yml

```
@@ -0,0 +1,5 @@
+minor_changes:
+- >-
+  aws_ec2 inventory - use the iam_role_arn configuration parameter to assume the role before trying to call DescribeRegions
+  if the regions configuration is not set and AWS credentials provided without enough privilege to perform the DescribeRegions action.
+  (https://github.com/ansible-collections/amazon.aws/issues/566)
```

```suggestion
bugfixes:
- >-
  aws_ec2 inventory - use the iam_role_arn configuration parameter to assume the role before trying to call DescribeRegions
  if the regions configuration is not set and AWS credentials provided without enough privilege to perform the DescribeRegions action.
  (https://github.com/ansible-collections/amazon.aws/issues/566).
```

in my understanding, this is a bugfix.